api/src/Entity/User.php

```diff
@@ -268,6 +270,23 @@ public function isEmployee(): bool
         return false;
     }
 
+    public function isAssociatedWithRepairer(int|string|Repairer $repairer): bool
+    {
+        if ($this->isBoss()) {
+            return !$this->repairers->filter(function (Repairer $repairerOfUser) use ($repairer) {
+                return $repairer instanceof Repairer ? $repairerOfUser->id === $repairer->id : $repairerOfUser->id == $repairer;
```

Comment:
```suggestion
                return $repairer instanceof Repairer ? $repairerOfUser->id === $repairer->id : $repairerOfUser->id === $repairer;
```

Reply:
Ok I understand, but in any case, it's better to use a strict checking. In your case, your function must accept only integer as argument, update your function like that: 
```
public function isAssociatedWithRepairer(int $id): bool
```
you can var dump the `$id` and see than the var type will be int.
After that change `==` to `===`

You can also if you want add a requirement for the route parameter like: 
```
requirements: ['repairer_id' => '\d+'],
```